dockstore-webservice/src/main/resources/migrations.test.add_test_tools.xml

```diff
@@ -0,0 +1,279 @@
+            <column name="path" value="/Dockerfile"/>
+            <column name="type" value="DOCKERFILE"/>
+        </insert>
+        <insert tableName="sourcefile">
+            <column name="id" valueNumeric="72"/>
+            <column name="content" value="testparameterstuff"/>
+            <column name="path" value="/test1.json"/>
+            <column name="type" value="CWL_TEST_JSON"/>
+        </insert>
+        <insert tableName="sourcefile">
+            <column name="id" valueNumeric="73"/>
+            <column name="content" value="moretestparameterstuff"/>
+            <column name="path" value="/test2.json"/>
+            <column name="type" value="CWL_TEST_JSON"/>
+        </insert>
+        <insert tableName="tag">
+            <column name="id" valueNumeric="70"/>
+            <column name="dirtybit" valueBoolean="false"/>
+            <column name="hidden" valueBoolean="false"/>
+            <column name="lastmodified" valueDate="2016-02-04 16:44:00.0"/>
+            <column name="name" value="master"/>
+            <column name="reference" value="refs/heads/master"/>
+            <column name="valid" valueBoolean="true"/>
+            <column name="verified" valueBoolean="false"/>
+            <column name="verifiedsource"/>
+            <column name="automated" valueBoolean="true"/>
+            <column name="cwlpath" value="/Dockstore.cwl"/>
+            <column name="dockerfilepath" value="/Dockerfile"/>
+            <column name="imageid" value="123456"/>
+            <column name="size" valueNumeric="108608297"/>
+            <column name="wdlpath" value="/Dockstore.wdl"/>
+        </insert>
+        <insert tableName="tag">
+            <column name="id" valueNumeric="71"/>
+            <column name="dirtybit" valueBoolean="false"/>
+            <column name="hidden" valueBoolean="true"/>
+            <column name="lastmodified" valueDate="2016-02-04 16:44:00.0"/>
+            <column name="name" value="master2"/>
+            <column name="reference" value="refs/heads/master"/>
+            <column name="valid" valueBoolean="true"/>
+            <column name="verified" valueBoolean="false"/>
+            <column name="verifiedsource"/>
+            <column name="automated" valueBoolean="true"/>
+            <column name="cwlpath" value="/Dockstore.cwl"/>
+            <column name="dockerfilepath" value="/Dockerfile"/>
+            <column name="imageid" value="123456"/>
+            <column name="size" valueNumeric="108608297"/>
+            <column name="wdlpath" value="/Dockstore.wdl"/>
+        </insert>
+        <insert tableName="tool">
+            <column name="id" valueNumeric="70"/>
+            <column name="author" value=""/>
+            <column name="defaultversion"/>
+            <column name="description" value=""/>
+            <column name="email" value=""/>
+            <column name="giturl" value="git@github.com:github.com/denis-yuen/test1.git"/>
```

Comment:
probably doesn't matter, but this should not be using a workflow in my personal repo